scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
> But it is quite an effort for tab completion of plugin names?

Yes, I think so too. We should not make things more complicated if we do not need to. In my opinion the run-time of `kdb list` on Linux looks already good. Would be nice if we could have similar performance on macOS too though.

> The other parts of tab completion shouldn't be effected, or are they?

Not as far as I can tell.